tools/sigclient/pkg/query/query.go

```diff
@@ -909,3 +910,130 @@ func RunQueryFromFileAndOutputResponseTimes(dest string, filepath string, queryR
 
 	log.Infof("RunQueryFromFileAndOutputResponseTimes: Query results written to CSV file: %v", queryResultFile)
 }
+
+func runBenchmarkQueryBatch(uuidList []string, wg *sync.WaitGroup, semaphore chan struct{}) {
+
+	defer wg.Done()
+	defer func() { <-semaphore }()
+	for _, uuid := range uuidList {
+
+		data := map[string]interface{}{
+			"state":         "query",
+			"searchText":    "ident=" + uuid,
+			"startEpoch":    "now-1d",
+			"endEpoch":      "now",
+			"indexName":     "*",
+			"queryLanguage": "Splunk QL",
+		}
+		evaluationType := "total"
+		var relation, expectedValue string
+		relation = "eq"
+		expectedValue = "1"
+		// create websocket connection
+		conn, _, err := websocket.DefaultDialer.Dial("ws://localhost:5122/api/search/ws", nil)
```

Comment:
Should this read the address from a command line arg so we can test on non-localhost setups?